src/plugins/code-loader.js

```diff
@@ -0,0 +1,29 @@
+// Your custom Docusaurus plugin
+
+const fs = require('fs');
+
+const plugin = (options) => {
+    const codeLoadRegex = /CODE_LOAD::([^\s]+)/g;
+
+    const replace = (str) => str.replace(codeLoadRegex, (match, filePath) => {
+        const fileContent = fs.readFileSync('./code_snippets/' + filePath, 'utf8');
+
+        // Split to only keep lines between "start_here" and "end_here"
+        const lines = fileContent.split("start_here").pop().split("end_here").shift();
```

Comment:
Will this pick match everything with `start_here` and `end_here` right?

Perhaps to make conflicts more unlikely, let's add some symbols? Like `<start_here>` and `<end_here>`?

Reply:
ah ok you got it in next commit :)